k8s/prod/common/sscs/sscs-bulk-scan.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: sscs-bulk-scan
+  namespace: sscs
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: sscs-bulk-scan
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: sscs-bulk-scan
+    version: 0.0.4
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "d6c3dd58-af16-425d-8c4a-f491e895e35d"
+      image: hmctspublic.azurecr.io/sscs/sscs-bulk-scan:latest
+      environment:
+        IDAM_URL: https://idam-api.platform.hmcts.net
+        IDAM_OAUTH2_REDIRECT_URL: https://sscs-case-loader-prod.service.core-compute-prod.internal
+      aadIdentityName: sscs
```

Comment:
this shouldn't be here, it's coming from values.yaml here:
https://github.com/hmcts/sscs-bulk-scan/blob/master/charts/sscs-bulk-scan/values.yaml#L4

Reply:
```suggestion
```